pkg/googlesheets/googleclient/googleclient.go

```diff
@@ -78,33 +78,50 @@ func (gc *GoogleClient) GetSpreadsheetFiles() ([]*drive.File, error) {
 }
 
 func createSheetsService(ctx context.Context, auth *Auth) (*sheets.Service, error) {
-	if auth.AuthType == "none" {
+	if len(auth.AuthType) == 0 {
+		return nil, fmt.Errorf("Missing AuthType setting")
+	}
+
+	if auth.AuthType == "key" {
 		if len(auth.APIKey) == 0 {
-			return nil, fmt.Errorf("Invalid API Key")
+			return nil, fmt.Errorf("Missing API Key")
```

Comment:
Go errors should start with a non-capital letter.
```suggestion
			return nil, fmt.Errorf("missing API Key")
```